.gitignore

```diff
@@ -1 +1,11 @@
 bfvmm/bin/cross/libc++.so
+.build/
+*.a
+bin/
```

Comment:
That should be an exception (and actually shouldn't happen at all). The contents of bin/ folders should be generated by the build and that's it.